### PR TITLE
Update `sandbox` keyword descriptions

### PR DESCRIPTION
**Pull request by NiedziolkaMichal**
### Description

I have added more details to most of the keywords available in the sandbox attribute. I have also made descriptions the same on `iframe` page and `CSP: sandbox` page.

### CODE REVIEW

**Comment by NiedziolkaMichal** [comment]
Do we need `The sandbox attribute is unsupported in Internet Explorer 9 and earlier.` note?

**Comment by sideshowbarker** [comment]
> Do we need `The sandbox attribute is unsupported in Internet Explorer 9 and earlier.` note?

No we don’t — it can be dropped

**Comment by NiedziolkaMichal** [comment]
Alright, it has been removed.

File: files/en-us/web/html/element/iframe/index.md [hunk 1]
```diff
@@ -96,26 +96,26 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
 
   - : Applies extra restrictions to the content in the frame. The value of the attribute can either be empty to apply all restrictions, or space-separated tokens to lift particular restrictions:
 
+    - `allow-downloads`: This value allows you to download files through an {{HTMLElement("a")}} or {{HTMLElement("area")}} element with the [download](/en-US/docs/Web/HTML/Element/a#attr-download) attribute, as well as through the navigation that leads to a download of a file. This works regardless of whether the user clicked on the link, or JS code initiated it without user interaction.
```
Comment (teoli2003) [inline files/en-us/web/html/element/iframe/index.md:99]: Nit (to keep the same sentence structure as below):
```suggestion
    - `allow-downloads`: Allows downloading files through an {{HTMLElement("a")}} or {{HTMLElement("area")}} element with the [download](/en-US/docs/Web/HTML/Element/a#attr-download) attribute, as well as through the navigation that leads to a download of a file. This works regardless of whether the user clicked on the link, or JS code initiated it without user interaction.
```

File: files/en-us/web/http/headers/content-security-policy/sandbox/index.md [hunk 2]
```diff
@@ -50,45 +50,35 @@ Content-Security-Policy: sandbox <value>;
 where `<value>` can optionally be one of the following values:
 
 - `allow-downloads`
-  - : Allows for downloads after the user clicks a button or link.
+  - : This value allows you to download files through an {{HTMLElement("a")}} or {{HTMLElement("area")}} element with the [download](/en-US/docs/Web/HTML/Element/a#attr-download) attribute, as well as through the navigation that leads to a download of a file. This works regardless of whether the user clicked on the link, or JS code initiated it without user interaction.
```
Comment (teoli2003) [inline files/en-us/web/http/headers/content-security-policy/sandbox/index.md:53]: Nit:
```suggestion
  - : Allows downloading files through an {{HTMLElement("a")}} or {{HTMLElement("area")}} element with the [download](/en-US/docs/Web/HTML/Element/a#attr-download) attribute, as well as through the navigation that leads to a download of a file. This works regardless of whether the user clicked on the link, or JS code initiated it without user interaction.
```

**Review by teoli2003 — APPROVED** [review]
2 nits that I am commiting.

**Pull request merged**